src/Application/FileProcessor.php

```diff
@@ -44,11 +44,14 @@ public function processFileWithRectorsToString(SplFileInfo $file, array $rectorC
         return $this->processFileToString($file);
     }
 
-    public function processFile(SplFileInfo $fileInfo): void
+    /**
+     * @return int|bool
+     */
+    public function processFile(SplFileInfo $fileInfo)
```

Comment:
Here `bool` is again preferred

Reply:
But as the `file_put_content` returns `int`, is correct to force bool?

Reply:
I think it is ok to retype, when you need to get answer:

-  `true` = file has changed
- `false` = file has not changed

Reply:
Ok, gonna use true and false